vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -272,25 +272,34 @@ func (r RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 	filters.Add("status", "running")
 	filters.Add("status", "paused")
 	filters.Add("status", "restarting")
-	containers, err := c.ContainerList(context.Background(), types.ContainerListOptions{
+
+	ctx, cancel := context.WithTimeout(context.Background(), dockerConnTimeout*time.Second)
+	defer cancel()
+	containers, err := c.ContainerList(ctx, types.ContainerListOptions{
 		All:    true,
 		Filter: filters,
 	})
 	if err != nil {
+		log.Errorf("ContainerList failed (err: %v)", err)
 		return err
 	}
 
 	log.Debugf("Found %d running or paused containers", len(containers))
 	for _, ct := range containers {
-		containerJSONInfo, err := c.ContainerInspect(context.Background(), ct.ID)
+		ctx_inspect, cancel_inspect := context.WithTimeout(context.Background(), dockerConnTimeout*time.Second)
+		defer cancel_inspect()
+		containerJSONInfo, err := c.ContainerInspect(ctx_inspect, ct.ID)
 		if err != nil {
 			log.Errorf("ContainerInspect failed for %s (err: %v)", ct.Names, err)
 			continue
 		}
 		log.Debugf("  Mounts for %v", ct.Names)
 
 		for _, mount := range containerJSONInfo.Mounts {
-			if mount.Driver == driverName {
+			// check if the mount location belongs to vmdk plugin
+			// managed plugin has mount source in format:
+			// '/var/lib/docker/plugins/{plugin uuid}/rootfs/mnt/vmdk/{volume name}'
+			if strings.Contains(mount.Source, mountRoot) {
```

Comment:
Can the check to compare with the driver name be retained? Is that changed with the container based plugin?